src/datastore/NedbDatastore.ts

```diff
@@ -214,6 +214,10 @@ export class NedbDatastore implements Datastore {
         return [];
     }
 
+    public async getPuppetMatrixUserBySlackId(teamId: string, slackId: string): Promise<null> {
+        return null;
```

Comment:
if we don't support nedb any longer do we need to keep stubbing these methods?

Reply:
Probably not. I may just chuck it out in another PR.

Reply:
Ah, but we use it for testing still...